### DISCUSSION TITLE
Add: TV Ciudad

### DISCUSSION THREAD

**Discussion by josepedro770** [iptv-org/community#471] [Show and tell]
### Channel Name

TV Ciudad

### Channel ID

TVCiudad.uy

### Country

Uruguay

### Language

Spanish

### Website

https://www.tvciudad.uy/

### Stream URL

https://vxlive.cdn.antel.net.uy/auth_V_TvCiudad,vxttoken=cGF0aFVSST0lMkZhdXRoX1ZfVHZDaXVkYWQlMkZ2eGZtdCUzRGhscyUyRiUyQSZleHBpcnk9MTYzOTc3OTk1NyZyYW5kb209bW1kWGRpSlE5QiZjLWlwPTE5MC42NC45Mi4xNDYmaWRfcmVmZXJlbmNpYT0maWRfZGlzcG9zaXRpdm89MCZpZF9mcm9udGVuZD0xMTk2JnB1YmxpY19pZD0yczZyMiwzOTE0ZjhkMzNmNjE4ZThkYzk3YTM4ZjJhZWYxY2U2ZjVhZGEwNGZjYTcxMDdkYjhjNTNhNDA1MmQ4ODZiZjJj/vxfmt=hls/h_6d57bc2902c05000e4e33be157085a38/var3000000/playlist.m3u8

**Comment by AntiPontifex** [comment]
Already included on the list: please close.

**Comment by freearhey** [comment]
https://vxlive.cdn.antel.net.uy/auth_V_TvCiudad,vxttoken=cGF0aFVSST0lMkZhdXRoX1ZfVHZDaXVkYWQlMkZ2eGZtdCUzRGhscyUyRiUyQSZleHBpcnk9MTYzOTc3OTk1NyZyYW5kb209bW1kWGRpSlE5QiZjLWlwPTE5MC42NC45Mi4xNDYmaWRfcmVmZXJlbmNpYT0maWRfZGlzcG9zaXRpdm89MCZpZF9mcm9udGVuZD0xMTk2JnB1YmxpY19pZD0yczZyMiwzOTE0ZjhkMzNmNjE4ZThkYzk3YTM4ZjJhZWYxY2U2ZjVhZGEwNGZjYTcxMDdkYjhjNTNhNDA1MmQ4ODZiZjJj/vxfmt=hls/h_6d57bc2902c05000e4e33be157085a38/var3000000/playlist.m3u8 (offline)
<img width="838" alt="image" src="https://user-images.githubusercontent.com/7253922/232222337-f897a4cf-ce4c-472c-8127-cb69d623f1d5.png">

**Comment by freearhey** [comment]
http://streamingprotv.com:8000/iptvkingpremiumultrax/Ig9L0fKOP2/199430
<img width="790" alt="image" src="https://user-images.githubusercontent.com/7253922/234129560-8fb086b3-12c2-42ac-9745-b21a9af64d22.png">

**Comment by kaikyoofers** [comment]
Another xtream link
